python/mxnet/gluon/rnn/rnn_layer.py

```diff
@@ -301,11 +301,11 @@ class RNN(_RNNLayer):
     --------
     >>> layer = mx.gluon.rnn.RNN(100, 3)
     >>> layer.initialize()
-    >>> input = mx.nd.random_uniform(shape=(5, 3, 10))
+    >>> input = mx.nd.random.uniform(shape=(5, 3, 10))
```

Comment:
Did you check the documentation for sparse namespace last time? I think some are not correct e.g. https://github.com/apache/incubator-mxnet/blob/master/python/mxnet/ndarray/sparse.py#L303

Reply:
Seems I missed that. Checked in now.